spark/v3.5/spark/src/main/java/org/apache/iceberg/spark/procedures/SparkProcedures.java

```diff
@@ -61,6 +61,7 @@ private static Map<String, Supplier<ProcedureBuilder>> initProcedureBuilders() {
     mapBuilder.put("create_changelog_view", CreateChangelogViewProcedure::builder);
     mapBuilder.put("rewrite_position_delete_files", RewritePositionDeleteFilesProcedure::builder);
     mapBuilder.put("fast_forward", FastForwardBranchProcedure::builder);
+    mapBuilder.put("compute_table_stats", ComputeTableStatsProcedure::builder);
```

Comment:
Please also document it under 
https://github.com/apache/iceberg/blob/main/docs/docs/spark-procedures.md

Reply:
Thanks @ajantha-bhat , 
I will create a follow up PR for the doc changes, just to keep this PR focussed on the procedure.